docs/source/write_first_app.rst

```diff
@@ -179,7 +179,9 @@ Let's run the application and then step through each of the interactions with th
 ``asset-transfer-basic/application-gateway-typescript`` directory, run the following command:
```

Comment:
```suggestion
``asset-transfer-basic/application-gateway-typescript`` directory, run the following:
```

Reply:
Looks good, thanks.  Can you just make this suggested change to the wording since it is now referring to more than one command?